README.md

```diff
@@ -24,6 +25,14 @@ dotenv -- bash -c 'RUST_LOG=debug forge create \
   --rpc-url $JSON_RPC \
   src/Weth9.sol:WETH9'
 
+# deploy test token
```

Comment:
We can delete this since we removed the SteezeToken :( haha

Reply:
removed! rip steezetoken